outputs.tf

```diff
@@ -57,8 +52,3 @@ output "ebs_kms_key_arn" {
   description = "KMS Key ARN used for EBS encryption"
   value       = module.kms_ebs.key_arn
 }
-
-output "node_termination_handler_sqs_arn" {
-  description = "ARN of the SQS queue used to handle node termination events"
-  value       = module.node_termination_handler_sqs.sqs_queue_arn
-}
```

Comment:
Shouldn't remove this. Output null if it's not enabled.

Reply:
creation of `node_termination_handler_sqs_arn` for default core nodes is removed that's why this output is removed. 

`essentials` submodule also will not create nth by default, if we are provisioning self_managed node groups using submodule, then we have to create sqs separately and pass the arn to the essentials submodule.

Reply:
should I create the sqs as part of `essentials` ?

it's created here, right now.

https://github.com/SPHTech-Platform/terraform-aws-eks/blob/8b355c29008eaae7baf636c86e3a0f0d93c4611b/node_groups.tf#L113

Reply:
Sure.